index.html

```diff
@@ -5,115 +5,151 @@
         <title>MY KIND OF BUSINESS BLOG - POLO360</title>
         <link rel="stylesheet" href="css/reset.css" />
         <link rel="stylesheet" type="text/css" href="css/style.css" />
+        <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
     </head>
     <body>
-        <header>
-            <div>
-                <a id="logo" href="index.html"><img src="img/logo.png"/></a>
-                <nav>
-                    <a href="https://mic.am/">HOME<span>Back to home</span></a>
-                    <a href="https://mic.am/training-program/">PRODUCTS<span>What we have for you</span></a>
-                    <a href="https://mic.am/projects/">SERVICES<span>Things we do</span></a>
-                    <a href="https://mic.am/blog/">BLOG<span>Follow our updates</span></a>
-                    <a href="https://mic.am/about-us/">CONTACT<span>Ways to reach us</span></a>
-                </nav>
-            </div>
+        <header class="header">
+            <a class="logoHeader" href="index.html"><img src="img/logo.png"/></a>
+            <nav class="menuHeader">
+                <a href="https://mic.am/"><span class="linkHeader">HOME</span><br /><span class="linkDescription">Back to home</span></a>
+                <a href="https://mic.am/training-program/"><span class="linkHeader">PRODUCTS</span><span class="linkDescription">What we have for you</span class="linkDescription"></a>
+                <a href="https://mic.am/projects/"><span class="linkHeader">SERVICES</span><span class="linkDescription">Things we do</span></a>
+                <a href="https://mic.am/blog/"><span class="linkHeader">BLOG</span><span class="linkDescription">Follow our updates</span></a>
+                <a href="https://mic.am/about-us/"><span class="linkHeader">CONTACT</span><span class="linkDescription">Ways to reach us</span></a>
```

Comment:
Use Visual Studio code prettier extension, to make you HTML document mor appealing and easy readable.